src/kOS/Suffixed/StageValues.cs

```diff
@@ -6,17 +6,21 @@
 using System.Collections.Generic;
 using System.Linq;
 using KSP.UI.Screens;
+using kOS.Module;
 
 namespace kOS.Suffixed
 {
     [kOS.Safe.Utilities.KOSNomenclature("Stage")]
     public class StageValues : Structure
     {
         private readonly SharedObjects shared;
+        private HashSet<global::Part> partHash = new HashSet<global::Part>();
+        private PartSet partSet;
```

Comment:
Is this a new thing from the base game API?  If so I like it.  I could think of a few other places we could use it - like mapping a name tag to a PartSet in KOSNameTag to make managing the data a bit cleaner.